shadho-test.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+#unset PYTHONPATH to ignore existing python installs.
+export PYTHONPATH=
+
+#activate the conda shell hooks without starting a new shell
+CONDA_BASE=$(conda info --base)
+. $CONDA_BASE/etc/profile.d/conda.sh
+
+#create conda environment for shadho
+conda create --name shadho-test -y
+conda activate shadho-test
+
+#install ndcctools and shadho
+conda install -c conda-forge ndcctools -y
+conda install pip -y	#shadho is installed through pip
+python -m pip install shadho
+
+#run work_queue_worker. This command is heuristic and based on shadho code.
+$CONDA_PREFIX/bin/work_queue_worker -M sin_ex-tphung --cores 1 --single-shot &
```

Comment:
This is going to pop up in the catalog daily.  Let's call it shadho-wq-packaging-test or something like that.

Reply:
Shadho automatically appends user name to the project name, so shadho-wq-packaging-test is renamed to shadho-wq-packaging-test-tphung, for example (A bit weird to let user name exposed).